src/index.js

```diff
@@ -98,7 +97,7 @@ function commentLine (line) {
 
 module.exports = function (source) {
   try {
-    const sourceByLine = source.split(os.EOL)
+    const sourceByLine = source.split('\n')
```

Comment:
I would suggest to use regular /[\r\n]+/g to detect end of lines on all systems

`const sourceByLine = source.split(/[\r\n]+/g)`